From 59c4725adc255ddfcd4696174a87b42bdb6f87a6 Mon Sep 17 00:00:00 2001
From: Felix Krull <f_krull@gmx.de>
Date: Wed, 29 Aug 2018 18:27:40 +0200
Subject: [PATCH 3/6] ostree-boot: weaken dracut Depends to Recommends

The systemd parts of the package could also be used for integration
with e.g. initramfs-tools.

Upstream-Status: Backport

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index dfa992e3..d2560bc9 100644
--- a/debian/control
+++ b/debian/control
@@ -151,11 +151,11 @@ Description: content-addressed filesystem for operating system binaries
 Package: ostree-boot
 Architecture: linux-any
 Depends:
- dracut,
  ostree (= ${binary:Version}),
  ${misc:Depends},
  ${shlibs:Depends},
 Recommends:
+ dracut,
  grub-efi-amd64 | grub-pc | extlinux | syslinux | u-boot,
 Breaks:
  ostree (<< ${binary:Version}),
-- 
2.17.1

